### PR TITLE
Polling tentacles over 443

### PR DESCRIPTION
**Pull request by JonPHardy57**
# About

This is a post about configuring polling tentacles to use the recognised port for https: traffic i.e. port 443.

# Release

It is live now and is aimed at Octopus Cloud customers.

# Blog image idea

A tentacle and Octopus cloud connected by "443" 

# Prerequisites

- [Y ] The draft is complete and this post is ready to be reviewed.
- [Y] I'm familiar with the [guide for blog content basics](https://www.octopus.design/932c0f1a9/p/901d2a-blog-content).
- [Y] I've checked the content using Grammarly and the [Hemingway app](https://hemingwayapp.com/) to remove typos, grammatical errors, and passive voice. 
- [Y] The PR build passes validation, and if it doesn't, I've checked the [common validations errors](https://www.octopus.design/932c0f1a9/p/901d2a-blog-content/t/817249) and none of those apply.
- [Y ] The images I've included follow the [rules for working with images](https://www.octopus.design/932c0f1a9/p/5061d7-working-with-images).
- [Y] I've added alt text for all the images I've included (alt text describes image to people unable to see it, 125 characters max)
- [N] I've drafted social media posts for Twitter and LinkedIn and will post them for review in #topic-social-media.
- [ Y] I've dropped a note in Slack in #topic-content-marketing to let Tegan Ali know my post is ready for review. (No need to fill out a content brief.)

## How to review this PR

I included a tag "Tentacles" which was not in the tag list. There was nothing in the list that covered off tentacles as far as I can see. It would seem to make sense to have Tentacles as a tag, but up to you. 

Given the conversation with Ella a week or so ago, I was not sure if social media posts were desired. So, I have not written any yet. Let me know if we should have them.

## Publication date

The capability is live, so this blog and video can be published now, if desired.

### CODE REVIEW

File: blog/2023-06/polling-tentacles-443/Index.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  - Product
+  - Octopus Cloud
+  - Trust and Security
+---
+
+In Octopus Deploy, deployment targets communicate with Octopus Cloud using Tentacles. We made an update so that Polling Tentacles can use port 443, instead of port 10943. 
+
+Port 10943 is an unassigned port in the [IANA port listing](https://www.iana.org/assignments/service-names-port-numbers/service-names-port-numbers.xhtml?&page=120), and therefore looks suspicious to some. 
+
+Port 443 often meets an organization’s firewall rules for outgoing encrypted traffic. This avoids lengthy approval processes to get authorization for firewall rule exceptions for port 10944.
```
Comment (TeganAli) [inline blog/2023-06/polling-tentacles-443/Index.md:21]:
```suggestion
Port 443, however, often meets an organization’s firewall rules for outgoing encrypted traffic. This avoids lengthy approval processes to get authorization for firewall rule exceptions for port 10944. 
```

File: blog/2023-06/polling-tentacles-443/Index.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+  - Product
+  - Octopus Cloud
+  - Trust and Security
+---
+
+In Octopus Deploy, deployment targets communicate with Octopus Cloud using Tentacles. We made an update so that Polling Tentacles can use port 443, instead of port 10943. 
+
+Port 10943 is an unassigned port in the [IANA port listing](https://www.iana.org/assignments/service-names-port-numbers/service-names-port-numbers.xhtml?&page=120), and therefore looks suspicious to some. 
+
+Port 443, however, often meets an organization’s firewall rules for outgoing encrypted traffic. This avoids lengthy approval processes to get authorization for firewall rule exceptions for port 10944.
```
Comment (TeganAli) [inline blog/2023-06/polling-tentacles-443/Index.md:21]: @JonPHardy57 just checking whether this should say exceptions or exemptions? Thanks :)

Comment (JonPHardy57) [reply blog/2023-06/polling-tentacles-443/Index.md:21]: @TeganAli - the industry phrasing seems to be "Firewall exceptions" and therefore it is probably clearer to delete the word "rule" so "firewall rule exceptions" becomes "firewall exceptions".  An example of the usage is here https://ntg.missouristate.edu/policies/FirewallExceptions.aspx. Many thanks Jon.

Comment (TeganAli) [reply blog/2023-06/polling-tentacles-443/Index.md:21]: Ok cool, thank you for clarifying, @JonPHardy57. I'll update the post accordingly :)

Comment (JonPHardy57) [reply blog/2023-06/polling-tentacles-443/Index.md:21]: Thank you.

**Review by TeganAli — APPROVED** [review]

**Pull request merged**